SonarQube.TeamBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -18,7 +18,7 @@ namespace SonarQube.TeamBuild.PreProcessor.Roslyn
 {
     public class RoslynAnalyzerProvider : IAnalyzerProvider
     {
-        public const string RoslynCSharpFormatName = "roslyn-config-cs";
+        public const string RoslynCSharpFormatName = "roslyn-cs";
```

Comment:
Miscellaneous: changed the profile name to match the one in the prototype C# plugin jar @dbolkensteyn provided.